java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/WeightActivity.java

```diff
@@ -99,21 +103,49 @@ private void putWeight(String value) {
 	}
 
 	private void renderWeights(List<Thing2> things) {
-		int count = 0;
+		int xPoint = 0;
```

Comment:
Should add check for null things list. Not sure if it is possible given our current flows, but if you do call with a null things list we will get npe below on the call to size